posthog/client.py

```diff
@@ -522,48 +535,121 @@ def get_feature_flag(
             self.distinct_ids_feature_flags_reported[distinct_id].add(feature_flag_reported_key)
         return response
 
+    def get_feature_flag_payload(
```

Comment:
this needs to go in `__init__.py` for the interface to be available.

Also add to `example.py` to test locally that everything works :)